src/serializer.jl

```diff
@@ -239,6 +239,16 @@ function save(p::AbstractPickle, io::IO, x::Vector)
   batch_appends(p, io, x)
 end
 
+function setitem(p::AbstractPickle, io::IO, x::Dict, k, v, state)
```

Comment:
There are no `k` and `v` to pass at this moment

Reply:
Fixed